.github/workflows/vast.yaml

```diff
@@ -599,15 +599,15 @@ jobs:
             cc: gcc-10
             cxx: g++-10
             dependencies-script-path: scripts/debian/install-dev-dependencies.sh
-            cmake-extra-flags: -DVAST_ENABLE_BUNDLED_CAF:BOOL=OFF
+            cmake-extra-flags:
           - os: macos-latest
             container: null
             name: macOS
             compiler: Clang
             cc: clang
             cxx: clang++
             dependencies-script-path: scripts/macOS/install-dev-dependencies.sh
-            cmake-extra-flags: -DVAST_ENABLE_BUNDLED_CAF:BOOL=ON
+            cmake-extra-flags:
```

Comment:
brew installed CAF is 0.18.5. I have no idea if there is a workaround to install 0.18.6. Maybe we can request some BUMP in brew?

Reply:
Can we build with 0.18.5+? Then we could also speed up the Debian CI and Docker image build by using the one that's available upstream.

If not then I can request a bump on Homebrew, but this would then be in a follow-up.

Reply:
We can for macOS. For debian we can download only 0.17.6 though

Reply:
Check. Makes sense.